testhelper/workspaceConfig/workspaceConfig.go

```diff
@@ -0,0 +1,28 @@
+package workspaceConfig
+
+import (
+	"html/template"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func CreateTempFile(t *testing.T, templatePath string, values map[string]string) string {
+	t.Helper()
+	tpl, err := template.ParseFiles(templatePath)
+	require.NoError(t, err)
+
+	tmpFile, err := os.CreateTemp("", "workspaceConfig.*.json")
+	require.NoError(t, err)
+	t.Cleanup(func() { _ = tmpFile.Close() })
```

Comment:
the file could be closed earlier, after this function completes with a defer statement, correct?